tests/fixtures/saas_fixtures.py

```diff
@@ -109,3 +227,29 @@ def mailchimp_account_email():
     return pydash.get(saas_config, "mailchimp.account_email") or os.environ.get(
         "MAILCHIMP_ACCOUNT_EMAIL"
     )
+
+
+@pytest.fixture(scope="function")
+def reset_saas_data(connection_config_saas, mailchimp_account_email) -> Generator:
```

Comment:
It would be helpful to indicate that this test is Mailchimp specific in the name.

Reply:
Changed fixture name to `reset_mailchimp_data`